### DISCUSSION TITLE
Support for 2FA on the backend login page

### DISCUSSION THREAD

**Discussion by hirisov** [wintercms/winter#774] [Ideas]
It would be nice to have 2FA option for the backend athentication for increased security.

There is an existing plugin for OctoberCMS for that:

https://octobercms.com/plugin/vdlp-twofactorauthentication

but the developers are only interesed in supporting OctoberCMS, so they will not make this plugin available for WinterCMS, and I think this should be a nice core feature anyway.

**Comment by LukeTowers** [comment]
I have detailed plans for MFA support in the core, just need to have the time to implement them. Basically will boil down to offering multiple "drivers" to support MFA over ToTP, SMS, Email (not really MFA but some people are convinced it is 🙄), WebAuthN, etc.

**Reply by hirisov** [reply]
Thant sounds like a great improvement, thanks for the feedback!
